codespell_lib/data/dictionary.txt

```diff
@@ -1450,6 +1450,7 @@ amerliorate->ameliorate
 amgle->angle
 amgles->angles
 amiguous->ambiguous
+aminator->animator, laminator,
```

Comment:
You say "aminator is a legit word but not frequently used in comparison to it being a typo." So this should be moved into rare too.

Ideally with a reference to where it's a real word.

Reply:
Aminator is used by netflix to create amazon machine images:

https://github.com/Netflix/aminator
https://netflixtechblog.com/ami-creation-with-aminator-98d627ca37b0

Reply:
Hmm, perhaps code dictionary then, good find @sebweb3r .

Reply:
ping @luzpaz.  If you don't want to do this I'd be happy to :)